### PR TITLE
add view_contact function and test

### PR DESCRIPTION
**Pull request by Ogollah**
*What does this PR do?*
creates view function and its unit test and correct a typo in delete contact unit test function

*Screenshot**
unitest

### CODE REVIEW

**Comment by ridgekimani** [comment]
The title is not supposed to be there on the description @Ogollah

File: contact.py [hunk 1]
```diff
@@ -19,8 +19,16 @@ def create_contact(self, name, phon_num):
...
+
+        #delete function
+    def delete_contact(self, contact):
         if contact in self.contact_list:
             self.contact_list.remove(contact)
-        return {'message':'Contact deleted!'}
\ No newline at end of file
+        return {'message':'Contact deleted!'}
+
+
```
Comment (ridgekimani) [inline contact.py]: Unnecessary spaces

File: test_contact.py [hunk 2]
```diff
@@ -11,10 +11,28 @@ def test_create_contact(self):
...
+
+
     def test_delete_contact(self):
         contact = Contacts()
-        response = contact.delet_contact(contact)
+        response = contact.delete_contact(contact)
         self.assertEqual(response['message'], 'Contact deleted!')
 
+
+
```
Comment (ridgekimani) [inline test_contact.py]: Unnecessary spaces

Comment (Ogollah) [reply test_contact.py]: Thank you for your comments

**Pull request merged**